test/dummy/app/views/layouts/application.html.haml

```diff
@@ -3,6 +3,8 @@
   %head
     %title Dummy
     %meta{:content => "width=device-width, initial-scale=1, maximum-scale=1, user-scalable=no", :name => "viewport"}/
+    %script{:src => "//ajax.googleapis.com/ajax/libs/jquery/2.1.3/jquery.min.js"}
```

Comment:
Main apps don't use jQuery 2 yet. I recommend to use the same version as the main_app here (to avoid future surprises).